.github/workflows/mvn-verify-check.yml

```diff
@@ -78,6 +89,32 @@ jobs:
           -Dmaven.javadoc.skip
           -Dmaven.scalastyle.skip=true
           -Drat.skip=true
+          
+  package-snapshotsOnly-aggregator:
+    needs: get-noSnapshot-versions-from-dist
+    strategy:
+      matrix: ${{ fromJSON(needs.get-noSnapshot-versions-from-dist.outputs.sparkSnapshotOnlyVersions) }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3 # refs/pull/:prNumber/merge
+
+      - name: Setup Java and Maven Env
+        uses: actions/setup-java@v3
+        with:
+          distribution: adopt
+          java-version: 8
+
+      - name: package aggregator check for snapshot builds
+        continue-on-error: true
```

Comment:
@gerashegalov - This doesn't seem to work as intended.  It shows the build as PASSED everytime.

Reply:
if this is the case we can simplify the PR by just using `SPARK_SHIM_VERSIONS_SNAPSHOTS_TAIL` at 
https://github.com/NVIDIA/spark-rapids/blob/058e684f9aa521d14c473ced9fb025d7001437b4/.github/workflows/mvn-verify-check.yml#L47

Reply:
All other builds are cancelled once spark340 build fails. So i think there is more to be looked into.

Reply:
did you try to disable fail-fast https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#handling-failures

Reply:
I don;t think fail-fast is happening when I set `continue-on-error: true`, all builds come out as passed even when spark-340 is failing. Could see build error in the log

Reply:
Correction: setting `fail-fast:false` explicitly seems to work.